client/js/widgets/button.js

```diff
@@ -385,7 +385,7 @@ export class Button extends Widget {
 
       if(a.func == 'LABEL') {
         setDefaults(a, { value: 0, mode: 'set', collection: 'DEFAULT' });
-        if([ 'set', 'dec', 'inc' ].indexOf(a.mode) == -1)
+        if([ 'set', 'dec', 'inc','append' ].indexOf(a.mode) == -1)
           problems.push(`Warning: Mode ${a.mode} will be interpreted as add.`);
```

Comment:
The default is 'set' in the call to `problems`. Looks like this line was copy-pasted from SELECT?